app/javascript/packages/webauthn/webauthn-verify-button-element.ts

```diff
@@ -58,19 +59,26 @@ class WebauthnVerifyButtonElement extends HTMLElement {
       this.setInputValue('client_data_json', result.clientDataJSON);
       this.setInputValue('signature', result.signature);
     } catch (error) {
-      if (!isExpectedWebauthnError(error)) {
+      if (!isExpectedWebauthnError(error, { isVerifying: true })) {
         trackError(error);
       }
 
+      if (isUserVerificationScreenLockError(error)) {
+        this.setInputValue('screen_lock_error', 'true');
+      }
+
       this.setInputValue('webauthn_error', error.name);
     }
 
     this.closest('form')?.submit();
   }
 
   setInputValue(name: string, value: string) {
-    const input = this.querySelector<HTMLInputElement>(`[name="${name}"]`)!;
+    const input = this.ownerDocument.createElement('input');
```

Comment:
I have a sneaking feeling of deja vu like I'd tried this in the past and there's a trap waiting for me, but I've yet to encounter it.

Reply:
> I have a sneaking feeling of deja vu like I'd tried this in the past and there's a trap waiting for me, but I've yet to encounter it.

Okay, I think I bumped into it with the [current build failure](https://gitlab.login.gov/lg/identity-idp/-/jobs/853278), which is that we some of our feature specs run through these screens without JavaScript enabled and expect to be able to set the input values directly, so they need to be present in the page.

https://github.com/18F/identity-idp/blob/4172a4601b5f21bcc425f58a577b91e9d8ce77c9/spec/support/features/webauthn_helper.rb#L121-L128

I don't love letting the tests guide our implementation like this, but it's also not too big an issue to work around.